test/intl_suggestors/intl_migrator_test.dart

```diff
@@ -1299,5 +1299,132 @@ void main() {
         expect(messages.messageContents(), expectedFileContent);
       });
     });
+
+    group('Ignore', () {
+      test('Ignore statement with ignore comment', () async {
+        final source = 'import \'package:over_react/over_react.dart\';\n'
+            '\n'
+            'mixin FooProps on UiProps {}\n'
+            '\n'
+            'UiFactory<FooProps> Foo = uiFunction(\n'
+            '  (props) {\n'
+            '    return (Dom.div())(\n'
+            '      \'testString1\',\n'
+            '      \'testString2\',\n'
+            '    );\n'
+            '  },\n'
+            '  _\$FooConfig, //ignore: undefined_identifier\n'
+            ');\n'
+            '\n'
+            'UiFactory<FooProps> Bar = uiFunction(\n'
+            '  (props) {\n'
+            '    // IGNORE STATEMENT - INTL\n'
+            '    return (Dom.div())(\n'
+            '      \'testString1\',\n'
+            '      \'testString2\',\n'
+            '    );\n'
+            '  },\n'
+            '  _\$FooConfig, //ignore: undefined_identifier\n'
+            ');\n'
+            '';
+        final output = 'import \'package:over_react/over_react.dart\';\n'
+            '\n'
+            'mixin FooProps on UiProps {}\n'
+            '\n'
+            'UiFactory<FooProps> Foo = uiFunction(\n'
+            '  (props) {\n'
+            '    return (Dom.div())(\n'
+            '      TestClassIntl.testString1,\n'
+            '      TestClassIntl.testString2,\n'
+            '    );\n'
+            '  },\n'
+            '  _\$FooConfig, //ignore: undefined_identifier\n'
+            ');\n'
+            '\n'
+            'UiFactory<FooProps> Bar = uiFunction(\n'
+            '  (props) {\n'
+            '    // IGNORE STATEMENT - INTL\n'
+            '    return (Dom.div())(\n'
+            '      \'testString1\',\n'
+            '      \'testString2\',\n'
+            '    );\n'
+            '  },\n'
+            '  _\$FooConfig, //ignore: undefined_identifier\n'
+            ');\n'
+            '';
+
+        await testSuggestor(
+          input: source,
+          expectedOutput: output,
+        );
+      });
+
+      test('Ignore line in chained objects', () async {
+        final source = '''
+        class Foo {
+          String str;
+          Foo(this.str);
+
+          String value() {
+            return str;
+          }
+        }
+
+        class Bar {
+          Foo newFoo(str) {
+            return Foo(str);
+          }
+        }
+
+        class Baz {
+          Bar newBar() {
+            return Bar();
+          }
+        }
+
+        class Biz {
+          static const message = "Message";
```

Comment:
As per chat, this test's setup only sets up the IntlMigrator, which isn't what changes constants.